### ISSUE TITLE
Change MC's name

### ISSUE THREAD

**Issue by Safirah** [rfassoc/libseven#2]
The users could be able to give MC any name they want, right now they are limited by the name of choice of the creator of the chatroom

**Comment by phantamanta44** [comment]
actually MC avatars and names are drawn from the player's twitter profile if they're logged in

**Comment by Safirah** [comment]
Yes indeed, but users who didn't login should be able to change their name as well, and even if a user logs in, they could be able to change their nick
